server/database/queries/getTimeSlots.js

```diff
@@ -0,0 +1,10 @@
+const dbconnection = require('../dbConnection');
```

Comment:
Not sure about this part, I'm going to run the app to see how it's working now

Reply:
I see, this works! You could also consider loading the public holidays from the server so that you have centralized control of them. But this can be a later enhancement. I'm thinking about if you were in the future to introduce the control panel for the administrators, for example, they should be able to control exactly which dates are not available to book.